doc/ifa/SONiC_InbandFlowAnalyzer_FeatureSpec_Ver1.1.md

```diff
@@ -0,0 +1,478 @@
+
+Ingress node matches an incoming flow based on the ACL configuration specified in an IFA flow and makes a copy of a sampled packet from the flow based upon the configured sampling rate. The copied packet has the IFA meta-data instructions and IFA header and IFA meta-header data added, and is then forwarded along the same path as the sampled packet. 
+
+At intermediate nodes, IFA probe packets are identified and IFA meta-data is inserted and forwarded to next hop. IFA meta-data consists of fields such as IFA probe packet's ingress port, egress port, ingress time-stamp and egress time-stamp etc.
+
+At egress node, IFA probe packets are identified and IFA meta-data is inserted. Egress node terminates IFA session and summarizes the telemetry data for the entire path, encapsulates the telemetry data as payload and sends it to collector.
+
+At all other non-IFA nodes, IFA sampled packet is forwarded as a normal packet in the network.
+
+***Example IFA multinode topology***
+
+![IFA multinode topology](images/IFA_Multinode_Topology.JPG)
+
+***Example CLI configuration***
+
+    ***Ingress node***
+
+        IFA configuration
+            config ifa device-id 2345
+            config ifa flow-name flow1 acl_ifa 1000
+
+        ACL configuration
+            "ACL_TABLE": {
+                "IFA": {
+                    "policy_desc" : "IFA Ingress Device Policy",
+                    "stage" : "INGRESS",
+                    "type" : "L3" ,
+                    "ports" : "Ethernet20"
+                    }
+            }
+
+            "ACL_RULE": {
+                "IFA|Flow1": {
+                    "PRIORITY" : "55",
+                    "IP_TYPE" : "ipv4any",
+                    "SRC_IP" : "10.10.0.26/32",
+                    "DST_IP" : "10.10.1.26/32",
+                    "PACKET_ACTION" : "int_insert"
+                }
+            }
+
+    ***Intermediate node***
+
+        IFA configuration
+            config ifa device-id 2346
+            config ifa device-type intermediate
+
+    ***Egress node***
+
+        IFA configuration
+            config ifa device-id 2347
+            config ifa collector-name collector1 ipv4 11.12.13.14 9070
+
+        ACL configuration
+            "ACL_TABLE": {
+                "IFA": {
+                    "policy_desc" : "IFA Egress Device Policy",
+                    "stage" : "INGRESS",
+                    "type" : "L3" ,
+                    "ports" : "Ethernet20"
+                    }
+            }
+
+            "ACL_RULE": {
+                "IFA|Flow1": {
+                    "PRIORITY" : "55",
+                    "IP_TYPE" : "ipv4any",
+                    "SRC_IP" : "10.10.0.26/32",
+                    "DST_IP" : "10.10.1.26/32",
+                    "PACKET_ACTION" : "int_delete"
```

Comment:
Why not use the command ``config ifa device-type egress`` to configure a device as egress?